http/servlet-undertow/src/test/java/io/quarkus/ts/http/undertow/UndertowMissingServletClassIT.java

```diff
@@ -0,0 +1,50 @@
+package io.quarkus.ts.http.undertow;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardCopyOption;
+
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.bootstrap.Service;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.services.QuarkusApplication;
+
+@QuarkusScenario
+public class UndertowMissingServletClassIT {
+    @QuarkusApplication(builder = WithMissingServlet.class)
+    static RestService app = new RestService()
+            .setAutoStart(false);
+
+    @Tag("https://github.com/quarkusio/quarkus/issues/44063")
+    @Test
+    void verifyUndertowIgnoreServletClassMissing() {
+        assertDoesNotThrow(() -> app.start(),
+                "The app should start without any issue");
+        app.logs().assertDoesNotContain("Local name must not be null");
+    }
+
+    public static void replaceForInvalidXML(Service service) {
```

Comment:
maybe this should had been in the resource builder; again, not important, just fyi